Libraries/WebSocket/WebSocketInterceptor.js

```diff
@@ -17,8 +17,8 @@ const originalRCTWebSocketSend = NativeWebSocketModule.send;
 const originalRCTWebSocketSendBinary = NativeWebSocketModule.sendBinary;
 const originalRCTWebSocketClose = NativeWebSocketModule.close;
 
-let eventEmitter: NativeEventEmitter;
-let subscriptions: Array<EventSubscription>;
+let eventEmitter;
```

Comment:
Why was `@flow` removed from your last commit?

Reply:
Could we leave this file untouched -- I realize the types could be dead but I'd prefer we do this in a separate PR so the intent is clearer

Reply:
> a separate PR

Done https://github.com/facebook/react-native/pull/32570